src/PointwiseFunctions/MathFunctions/Gaussian.cpp

```diff
@@ -102,11 +107,33 @@ void Gaussian<1, Fr>::pup(PUP::er& p) {
   p | center_;
 }
 
+template <typename Fr>
+bool Gaussian<1, Fr>::operator==(const MathFunction<1, Fr>& other) const {
+  const auto* derived_other = dynamic_cast<const Gaussian<1, Fr>*>(&other);
+  if (derived_other) {
```

Comment:
The implicit conversion from ptr to bool is a bit confusing on first glance. Could you make this explicit like 
```cpp
if (derived_other != nullptr)
```

Same for the other functions you added.

Reply:
done

Reply:
Missed this one. There are 2 in this file.